src/components/Sections/Personal/Bookmarks/Bookmarks/helper.js

```diff
@@ -0,0 +1,31 @@
+import { CT_DAILY_LESSON, CT_LESSON_PART, CT_LIKUTIM, CT_SOURCE } from '../../../../../helpers/consts';
+import { cuPartNameByCCUType } from '../../../../../helpers/utils';
+
+export const buildTitleByUnit = (cu, t, getPathByID) => {
+
+  if (!cu) return '';
+
+  const { content_type, film_date, collections, name } = cu;
+
+  if (content_type === CT_LESSON_PART) {
+    const ctLabel = t(`constants.content-types.${CT_DAILY_LESSON}`);
+    const fd      = t('values.date', { date: film_date });
+    return `${ctLabel} (${fd})`;
+  }
+
+  if (content_type === CT_SOURCE) {
+    const path        = getPathByID(cu.id)?.map(x => x.name);
+    const articleName = path.splice(-1);
+    return `${articleName} ${path.join('. ')}`;
+  }
+
+  if (content_type === CT_LIKUTIM) {
+    return name;
+  }
+
+  const collection = Object.values(collections)[0];
+  const part       = Number(collection?.ccuNames[cu.id]);
```

Comment:
Are you sure this is a number here?
What happens when collection does not exist?